homeassistant/components/freebox/router.py

```diff
@@ -30,6 +31,26 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 
+class FixedConnection(Connection):
+
+    lte_configuration_data_schema = {
+        'enabled': True
+    }
+
+    async def get_lte_config(self):
+        '''
+        Get lte connection configuration
+        '''
+        return await self._access.get('connection/lte/config/')
+
+    async def set_lte_config(self, lte_configuration_data=lte_configuration_data_schema):
+        '''
+        Update lte connection configuration
+        
+        /!\ Requires system config permission. Go to {FreeboxURL}#Fbx.os.app.settings.Accounts and allow the permission manually.
+        '''
+        await self._access.put('connection/lte/config/', lte_configuration_data)
+
```

Comment:
This is aleady in the lib: https://github.com/stilllman/aiofreepybox/blob/aiofreepybox/aiofreepybox/api/connection.py

You should not override the lib, use it as it is.